ansible/roles/operator-pipeline/templates/openshift/pipelines/operator-hosted-pipeline.yml

```diff
@@ -146,8 +146,32 @@ spec:
         pipeline service account registry tokens.
   tasks:
 
+    # extract the PR number from the PR URL
+    - name: get-pr-number
+      taskRef:
+        name: get-pr-number
+      params:
+        - name: pipeline_image
+          value: "$(params.pipeline_image)"
+        - name: git_pr_url
+          value: "$(params.git_pr_url)"
+
+    # acquire/lease the resource to resolve the conflict of concurrent pipelineruns
+    - name: acquire-lease
+      runAfter:
+        - get-pr-number
+      taskRef:
+        name: acquire-lease
+      params:
+        - name: pipeline_image
+          value: "$(params.pipeline_image)"
+        - name: lease-name
+          value: &leaseName "community-hosted-pipeline-pr-$(tasks.get-pr-number.results.git_pr_number)"
```

Comment:
Since we are acquiring lease for operator hosted pipeline, should not we use a different name here?

Reply:
Thanks, good catch. Changed to `operator-hosted-pipeline...`